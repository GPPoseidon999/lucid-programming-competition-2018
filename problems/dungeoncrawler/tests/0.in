5
Diamond Ruby
DarkerTombs BlueKey 1 BrownKey
LonelyMazes BrownKey 0
GraniteLair Diamond 2 BlueKey RedKey
DreadDelves Ruby 1 BrownKey
BoilingPits RedKey 1 BrownKey