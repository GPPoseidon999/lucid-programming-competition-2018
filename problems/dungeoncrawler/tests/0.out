lonelymazes dreaddelves boilingpits darkertombs granitelair